docs/shared/migration/from-turborepo.md

```diff
@@ -0,0 +1,189 @@
+
+| **Task Configuration:**         |                                                                                                   |
+| ------------------------------- | ------------------------------------------------------------------------------------------------- |
+| `extends`                       | N/A. The project configurations will always extend the `targetDefaults` defined in `nx.json`.     |
+| `pipeline[task].dependsOn`      | [Same syntax](/reference/project-configuration#dependson).                                        |
+| `pipeline[task].dotEnv`         | Define [file `inputs`](/reference/project-configuration#filesets)                                 |
+| `pipeline[task].env`            | Define [env `inputs`](/reference/project-configuration#env-variables)                             |
+| `pipeline[task].passThroughEnv` | N/A. See [Defining Environment Variables](/recipes/tips-n-tricks/define-environment-variables)    |
+| `pipeline[task].outputs`        | [Same syntax](/reference/project-configuration#outputs).                                          |
+| `pipeline[task].cache`          | Define in the [`nx.json` `cacheableOperations` property](/reference/nx-json#tasks-runner-options) |
+| `pipeline[task].inputs`         | [Same syntax](/reference/project-configuration#filesets).                                         |
+| `pipeline[task].outputMode`     | Use the [`--output-style` command line flag](/packages/nx/documents/run-many#output-style)        |
+| `pipeline[task].persistent`     | N/A.                                                                                              |
+
+## Command Equivalents
+
+|                             |                                                                                                                                                                                         |
+| --------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `turbo run test lint build` | [`nx run-many -t test lint build`](/packages/nx/documents/run-many)                                                                                                                     |
+| `--cache-dir`               | Set in [`nx.json` under `tasksRunnerOptions.default.options.cacheDirectory`](/reference/nx-json#tasks-runner-options)                                                                   |
+| `--concurrency`             | [`--parallel`](/packages/nx/documents/run-many#parallel)                                                                                                                                |
+| `--continue`                | [Use `--nx-bail`](/packages/nx/documents/run-many#nx-bail)                                                                                                                              |
```

Comment:
`--nx-bail` is the inverse though. Turbo by default stops in case of an error as far as I understood from their docs and you can change that by using the `--continue` flag. `--nx-bail` does the opposite, it allows to stop in case of an error.
_(as a side-note...for a non-native English speaker "bail" is a weird word..)_